templates/init.tpl

```diff
@@ -1,5 +1,8 @@
 #!/bin/sh
 
+# chkconfig: 2345 60 20
+# description: ${NAME}
```

Comment:
The shell variable `NAME` is defined below and also it's not available in the comments. Can you please change this line to use the template variable `{{name}}`?

```
# description: {{name}}
```